src/ngraph/runtime/gpu/gpu_emitter.cpp

```diff
@@ -800,6 +800,245 @@ cudnnSetOpTensorDescriptor(opTensorDesc,
                 writer.block_end();
                 return;
             }
+
+            template <>
+            void GPU_Emitter::EMITTER_DECL(ngraph::op::Pad)
+            {
+                auto pad = static_cast<const ngraph::op::Pad*>(node);
+                writer.block_begin("  // " + node->get_name());
```

Comment:
We have removed all of the trailing comments for block_begin() because we have put a comment description of the function prior to emitting the op. Look at CPU emitted code to see what it looks like. Comments in emitted function makes Common Function Elimination more difficult.

Reply:
Really? The comments screw the compiler up when optimizing common expressions? That almost seems fragile! I like how the CPU emitted code looks. In some cases we still need the {} for scope. I will look into how this is implemented on the CPU side.